Dockerfile

```diff
@@ -1,57 +1,53 @@
-# syntax=docker/dockerfile:1
+# Create pipenv image to convert Pipfile to requirements.txt
+FROM python:3.9-slim as pipenv
 
-# Set base image
+# Copy Pipfile and Pipfile.lock
+COPY Pipfile Pipfile.lock ./
+
+# Install pipenv and convert to requirements.txt
+RUN pip3 install --no-cache-dir --upgrade pipenv; \
+    pipenv requirements > requirements.txt
+
```

Comment:
This isolates the pipenv to pip requirements.txt conversion into it's own layer so it can be discarded since we will not be using pipenv in the final image.

We copy out requirements.txt since that's all we need from this image